src/app_service/service/message/dto/message.nim

```diff
@@ -64,6 +64,8 @@ type QuotedMessage* = object
   contentType*: ContentType
   deleted*: bool
   discordMessage*: DiscordMessage
+  albumImages*: seq[string]
+  albumImagesCount*: int
```

Comment:
Maybe I'm missing something, but I don't see where you're setting those values. I see you use them, but never setting them, so maybe they are not really used?
As far as I can see, all the magic happens in `setQuotedMessageImages`

Reply:
Hum, let's see :
the model is exposed with `albumImages` and `albumImageCount` from the view perspective.
And in the controller, an item has `quotedMessageAlbumMessageImages` and `quotedMessageAlbumImagesCount` which are mapped to `quotedMessage.albumImages` and `quotedMessage.albumImageCount`.
But let me have a second look...